Source/JavaScriptCore/interpreter/Interpreter.cpp

```diff
@@ -392,6 +392,11 @@ class GetStackTraceFunctor {
 
     IterationStatus operator()(StackVisitor& visitor) const
     {
+        if (auto* codeBlock = visitor->codeBlock()) {
+            if (codeBlock->ownerExecutable()->implementationVisibility() != ImplementationVisibility::Public)
+                return IterationStatus::Continue;
+        }
+
```

Comment:
Move this after `m_framesToSkip`'s clause. This skip number is internal one, so including private one too makes writing it easier.